server/verify_test.go

```diff
@@ -1065,3 +1117,139 @@ func TestVerifyAttestationWithSevSnp(t *testing.T) {
 		})
 	}
 }
+
+func TestVerifyAttestationWithTdx(t *testing.T) {
```

Comment:
AIUI, you are trying to write a TPM2 event log out below. Consider separating these out into helper functions, as that really helps the readability.